tensorflow_privacy/privacy/membership_inference_attack/membership_inference_attack.py

```diff
@@ -170,6 +171,48 @@ def run_attacks(attack_input: AttackInputData,
       privacy_report_metadata=privacy_report_metadata)
 
 
+def run_seq2seq_attack(attack_input: Seq2SeqAttackInputData,
+                       privacy_report_metadata: PrivacyReportMetadata = None,
+                       balance_attacker_training: bool = True) -> AttackResults:
+  """Runs membership inference attacks on a seq2seq model.
+
+  Args:
+    attack_input: input data for running an attack
+    privacy_report_metadata: the metadata of the model under attack.
+    balance_attacker_training: Whether the training and test sets for the
+          membership inference attacker should have a balanced (roughly equal)
+          number of samples from the training and test sets used to develop
+          the model under attack.
+
+  Returns:
+    the attack result.
+  """
+  attack_input.validate()
+  attacker = models.LogisticRegressionAttacker()
+
+  prepared_attacker_data = models.create_seq2seq_attacker_data(
+      attack_input, balance=balance_attacker_training)
+
+  attacker.train_model(prepared_attacker_data.features_train,
+                       prepared_attacker_data.is_training_labels_train)
+
+  # Run the attacker on (permuted) test examples.
+  predictions_test = attacker.predict(prepared_attacker_data.features_test)
+
+  # Generate ROC curves with predictions.
+  fpr, tpr, thresholds = metrics.roc_curve(
+      prepared_attacker_data.is_training_labels_test, predictions_test)
+
+  roc_curve = RocCurve(tpr=tpr, fpr=fpr, thresholds=thresholds)
+
+  attack_results = [SingleAttackResult(
+      slice_spec=_get_slice_spec(attack_input),
```

Comment:
Copying this comment: "Are slices actually supported by this type of attack?

If not, let's add that to the doc and return an error if the user has specified a slice"

Reply:
No, slices aren't supported by this type of attack. The default value of SlicingSpec was "all" so I included it in the attack result 😅. 

> If not, let's add that to the doc and return an error if the user has specified a slice.

Sure, I will update the PR with the specified changes.